conans/client/rest/rest_client_v2.py

```diff
@@ -37,10 +37,10 @@ def _get_file_list_json(self, url):
         data["files"] = list(data["files"].keys())
         return data
 
-    def _get_remote_file_contents(self, url):
+    def _get_remote_file_contents(self, url, cache):
```

Comment:
```suggestion
    def _get_remote_file_contents(self, url, use_cache):
```

Can you rename it to `use_cache`?